config/environments/production.rb

```diff
@@ -72,17 +65,17 @@
   # Send deprecation notices to registered listeners.
   config.active_support.deprecation = :notify
 
-  # Use default logging formatter so that PID and timestamp are not suppressed.
-  config.log_formatter = ::Logger::Formatter.new
-
-  # Use a different logger for distributed setups.
-  # require 'syslog/logger'
-  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
-
-  if ENV["RAILS_LOG_TO_STDOUT"].present?
-    logger           = ActiveSupport::Logger.new(STDOUT)
+  # Logging configuration
+  if ENV['LOGSTASH_ENABLE'] == 'true'
```

Comment:
Given that we look up an env var called `LOGSTASH_ENABLE` here, does it make sense for the implementation we call to have a generic name like `Logging`?

Reply:
Good point, I've changed it to the slightly more specific LogstashLogging.